packages/story-editor/src/app/pageDataUrls/pageDataUrlsProvider.tsx

```diff
@@ -16,52 +16,48 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
+import type { Page } from '@googleforcreators/elements';
 import { useMemo, useCallback, useState } from '@googleforcreators/react';
+import type { PropsWithChildren } from 'react';
 
 /**
  * Internal dependencies
  */
 import useIdleTaskQueue from '../../utils/useIdleTaskQueue';
 import storyPageToDataUrl from '../pageCanvas/utils/storyPageToDataUrl';
+import type { PageDataUrlsState, PageDataUrlsActions } from '../../types';
 import Context from './context';
 
-/**
- * @typedef {import('@googleforcreators/elements').Page} Page
- */
-
-function PageDataUrlProvider({ children }) {
-  const [dataUrls, setDataUrls] = useState({});
+function PageDataUrlProvider({ children }: PropsWithChildren<unknown>) {
+  const [dataUrls, setDataUrls] = useState<PageDataUrlsState['dataUrls']>({});
```

Comment:
Not a big fan of this notation here. I'd rather we just made separate types for these and used those types in the state as well.

Reply:
(also because the state only contains this one single prop)